CHANGELOG.md

```diff
@@ -2,6 +2,33 @@
 
 ## Unreleased
 
+## 0.7.0 - 2020-08-02
+
+NOTE: This release contains non-backward-compatible breaking SDK changes
+
+- Added an InMemoryMetricExporter
+- Added a toBuilder method to Labels
+- Added some semantic attribute constants
+- New ZPages extension module with TraceZ and TraceConfigZ pages implemented
+- Some overloads added for setting the parent Context
+- Some performance improvements in HttpTraceContext implementation
+- Removed null checks from the Trace APIs
+- The bare API will no longer generate Trace and Span IDs when there is no parent trace context.
+- Null Strings are no longer valid keys for Attributes
+- BREAKING CHANGE: The Sampler API was changed
```

Comment:
Does it make sense to section out breaking changes and bugfixes?

Reply:
I'm not opposed to that. I had put it in temporal order, because I'm lazy, but I'm fine with rearranging it. :)